pyproject.toml

```diff
@@ -22,6 +22,7 @@ src = ["src/**/*.pyx"]
 #ANSIBLE_PYLIBSSH_TRACING = "1"
 #CFLAGS = "-DCYTHON_TRACE=1 ${CFLAGS}"
 LDFLAGS = "-lssh ${LDFLAGS}"
+CFLAGS="-I /usr/local/include ${CFLAGS}"
```

Comment:
This shouldn't be a part of a spelling PR.

Reply:
Of course not. A branching reference error

Reply:
Looks like fixing this is the only blocker here.